pywbemtools/pywbemcli/_common.py

```diff
@@ -272,6 +272,7 @@ def pick_one_from_list(context, options, title):
         context.spinner_stop()
 
     click.echo(title)
+    index = 0  # Avoid pylint undefined-loop-variable
```

Comment:
This pylint should no longer be occuring since I changed to not use the index outside the loop and added max_option yesterday (approved and committed)

Reply:
DONE - removed it.